econml/score/ensemble_cate.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+
+
+class EnsembleCateEstimator:
+
+    def __init__(self, *, cate_models, weights):
+        self.cate_models = cate_models

```

Comment:
We need to store the fitted models. Cloning has the certificate that it only copies the parameters. 

We could deep copy? What do you think?

Reply:
Good point - just add a comment that we're explicitly not cloning because we need the fitted models.  Perhaps also add a note in the docstring that references to the live models are stored, so that e.g. refitting would affect the results of the methods on this class.